librosa/filters.py

```diff
@@ -703,6 +715,11 @@ def cq_to_chroma(n_input, bins_per_octave=12, n_chroma=12,
         If True, the first chroma bin will start at 'C'
         If False, the first chroma bin will start at 'A'
 
+    dtype : np.dtype
+        The data type of the output basis.
+        By default, uses 32-bit floating point.
```

Comment:
By default, uses a single-precision floating-point format, i.e. np.float32.
A double-precision format corresponds to dtype=np.float64